exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -19,3 +19,28 @@
  * random (generated in function): 40
  * output: "Sum with value 70 isd left in 30 from number 100"
  */
+
+/**
+ * #9 :: Export JS function "getDistanceMessageFromSumTo100" to generate a random number (in between 0-100) and sum it to given number.
+ *    - if sum result exceeds 100, then return message "Sum with value N exceeds in N from number 100"
+ *    - otherwise, return message "Sum with value N is left in N from number 100"
+ * @param {number} value - The input number to add to the random number.
+ * @returns {string} - The message indicating the relationship between the sum and 100.
+ */
+const getDistanceMessageFromSumTo100 = function (value) {
+    if (typeof value !== 'number' || value < 0) {
+      throw new Error("Input must be a non-negative number.");
+    }
+  
+    const random = Math.floor(Math.random() * 101);
+    const sum = value + random;
+  
+    if (sum > 100) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (sum > THRESHOLD) {}
```

- [ ] Extract number to constant